tests/e2e/flowtest/test_base_operation_node_drain.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import logging
+from time import sleep
+
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.pytest_customization.marks import aws_platform_required, vsphere_platform_required
+from ocs_ci.ocs import node, constants
+from ocs_ci.framework.testlib import E2ETest, tier2, config, ignore_leftovers
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import storage_cluster
+from tests.e2e.flowtest import flowtest_helpers
+from tests.e2e.flowtest.flowtest_helpers import create_pvc_delete, \
+    obc_put_obj_create_delete, FlowOperations
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier2
+class TestFlowBasedTests(E2ETest):
+    """
+    Tests Story/Flow based test scenarios
+
+    """
+    @vsphere_platform_required
+    @aws_platform_required
+    @pytest.mark.polarion_id("OCS-xsadsa")
+    def test_base_operation_node_drain(
+        self, node_drain_teardown, node_restart_teardown, pgsql_factory_fixture,
+        project_factory, multi_pvc_factory, mcg_obj, bucket_factory
+    ):
+        """
+        Test covers following flow operations while running workloads in the background:
+        1. Node drain
+        2. Add capacity
+        3. Node reboot
+        4. Node n/w failure
+
+        """
+        logger.info("Starting IO operations in Background")
+        project = project_factory()
+
+        bg_handler = flowtest_helpers.BackgroundOps()
+        executor_run_bg_ios_ops = ThreadPoolExecutor(max_workers=3)
+
+        logging.info("Started object IOs in background")
+        obc_ios = executor_run_bg_ios_ops.submit(
+            bg_handler.handler, obc_put_obj_create_delete, mcg_obj, bucket_factory, iterations=50
+        )
+        logging.info("Started pvc create and delete in background")
+        pvc_create_delete = executor_run_bg_ios_ops.submit(
+            bg_handler.handler,
+            create_pvc_delete,
+            multi_pvc_factory,
+            project,
+            iterations=100
+        )
```

Comment:
move logging.info("Started pvc create and delete in background") to after this line

Reply:
ack